src/components/Mission.js

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+
+import { joinMission, leaveMission } from '../redux/missions/missionsSlice';
+
+const Button = ({ isJoined, missionId }) => {
+  const dispatch = useDispatch();
+
+  if (isJoined) {
+    return (
+      <button
+        className="btnm failure__button"
+        type="button"
+        onClick={() => dispatch(leaveMission(missionId))}
+      >
+        Leave Mission
+      </button>
+    );
+  }
```

Comment:
- [ ] @team great work on your mission page , however the active member background color doesn't match the design given, below is a comparison for better understanding:

Design  Given        |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/224473139-a279b765-9cdf-468f-b2a4-b6957e92bd05.png) |  ![image](https://user-images.githubusercontent.com/61408860/224473145-b7ea2e7a-ebd3-4391-b436-15c8efb016ad.png)